projects/storefrontlib/shared/components/generic-link/generic-link.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import {
+  HTTP_PROTOCOL_REGEX,
+  MAILTO_PROTOCOL_REGEX,
+  TEL_PROTOCOL_REGEX,
+} from './generic-link.models';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class GenericLinkService {
```

Comment:
This service has a very specialized responsibility - being an extension point for the `GenericLinkComponent`. IMHO the service name `GenericLinkService` in `@spartacus/storefront` shounds a bit to general.
What do you think about naming it to `GenericLinkComponentService` so it corresponds directly to the `GenericLinkComponent`?